scripts/ci/install_deps.sh

```diff
@@ -3,23 +3,30 @@
 # exit when any command fails
 set -ex
 
+
 if [[ $TEST_SUITE = "Kubernetes"  ]]; then
-  # conntrack is required by Minikube
-  sudo apt-get update
-  sudo apt-get install -y conntrack
 
   # Download kubectl, which is a requirement for using minikube.
   curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v"${KUBERNETES_VERSION}"/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
 
-  # Download Helm
+  sudo apt-get update
+
+  # Make test dir to store test results
+  mkdir -p "${TEST_RESULTS}"
+
+  # conntrack is required by Minikube
+  sudo apt-get install -y conntrack
+
+  curl -sSL https://github.com/gotestyourself/gotestsum/releases/download/v"${GOTESTSUM_VERSION}"/gotestsum_"${GOTESTSUM_VERSION}"_linux_amd64.tar.gz | sudo tar -xz -C /usr/local/bin gotestsum
+
   wget https://get.helm.sh/helm-"${HELM_VERSION}"-linux-amd64.tar.gz -O /tmp/helm.tar.gz
   tar xzf /tmp/helm.tar.gz -C /tmp --strip-components=1 && chmod +x /tmp/helm && sudo mv /tmp/helm /usr/local/bin
 
   # Download minikube.
   curl -Lo minikube https://storage.googleapis.com/minikube/releases/v"${MINIKUBE_VERSION}"/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
 
   # start minikube
-  minikube start --kubernetes-version=v"${KUBERNETES_VERSION}" --memory=max --cpus=max
+  minikube start --vm-driver=none --kubernetes-version=v"${KUBERNETES_VERSION}"
```

Comment:
Although it has some advantages, the driver `none` is not recommended and has a number of [prerequisites](https://minikube.sigs.k8s.io/docs/drivers/none/#requirements) that needs to be satisfied. I couldn't get it working with newer K8s versions before. 

In NuoDB Helm charts, we needed to [lower the resource limits for the database containers](https://github.com/nuodb/nuodb-helm-charts/blob/master/scripts/ci/install_deps.sh#LL33C1-L47C4) to make it work with _docker_ driver.